backend/app/src/main/java/com/moh/phlat/backend/service/ProcessDataServiceImpl.java

```diff
@@ -24,4 +27,91 @@ public List<ProcessData> getProcessDataWithMessages(Long controlTableId, String
         }
 
     }
+
+    public List<ReportSummary> getReportSummary(Long controlTableId) {
```

Comment:
at all places, the name should be business specific. Could we check with business on what name we can give it to this report. Then we need to change at all places. 
1. URL
2. controller method name
3. service method name
4. class names
5. Any other place it is being used.

Reply:
I keep this mind for future changes. We will break the frontend if we change the endpoint now.